### DISCUSSION TITLE
App not compatible with device

### DISCUSSION THREAD

**Discussion by jrodbrown58** [JunkFood02/Seal#348] [Q&A] [answered]
Tried installing onto my OnePlus 7 Pro running A13 but keep receiving app incompatible errors. Am I doing something wrong? Or is there a particular reason why it's not installing? Thank you!

**Comment by JunkFood02** [comment, marked as answer]
Did you pick the right apk(arm64-v8a) to install?

**Comment by jrodbrown58** [comment]
That was the problem, I wasn't sure which one to download and I even tried
to download from F-Droid and still had the problem. so i went back to
Github and tried another version and it worked. Thank you!
